pkg/operator/ceph/cluster/osd/config/config.go

```diff
@@ -97,3 +100,31 @@ func convertToIntIgnoreErr(raw string) int {
 
 	return val
 }
+
+// A DriveGroupBlob is a simple JSON blob defining a Ceph Drive Group. Drive Group blobs are passed
+// to ceph-volume for node disk configuration.
+type DriveGroupBlob string
+
+// DriveGroupBlobs is a mapping from Ceph Drive Group names to JSON blobs of Drive Group specs.
+type DriveGroupBlobs map[string]string
+
+// MarshalAsDriveGroupBlobs converts a Ceph CRD <Drive Group Name> => <Drive Group Spec> mapping
+// into a JSON-marshalled <Drive Group Name> => <Drive Group JSON blob> mapping.
+func MarshalAsDriveGroupBlobs(spec cephv1.DriveGroupsSpec) (string, error) {
```

Comment:
This method is only needed in the parent package. How about moving it to that package so it can just be a local method?